src/cheri/attributes.adoc

```diff
@@ -24,57 +24,42 @@ endif::support_varxlen[]
 :cheri_priv_debug_trig: Sdytrig
 
 // 32-bit encodings added by RVY
-:rvyi_ext_name:         Zyi
-:rvyi_file_name:        {rvyi_ext_name}_insns_table_body
-:rvyi_xref:             <<{rvyi_file_name}, {rvyi_ext_name}>>
+:rvy_file_name:            {cheri_base_ext_name}_insns_table_body
 // 32-bit encoding for YSENTRY
-:rvyi_sentry_ext_name:  Zys
-:rvyi_sentry_file_name: {rvyi_sentry_ext_name}_insns_table_body
-:rvyi_sentry_xref:      <<{rvyi_sentry_file_name}, {rvyi_sentry_ext_name}>>
+:rvy_sentry_ext_name:      Zys
+:rvy_sentry_file_name:     {rvy_sentry_ext_name}_insns_table_body
 // 32-bit encodings modified by RVY
-:rvyi_mod_ext_name:     Zyi_mod
-:rvyi_mod_file_name:    {rvyi_mod_ext_name}_insns_table_body
-:rvyi_mod_xref:         <<{{rvyi_mod_file_name}, {rvyi_mod_ext_name}>>
+:rvy_i_mod_ext_name:       RVI ({cheri_base_ext_name} modified behavior)
+:rvy_i_mod_file_name:      {cheri_base_ext_name}_I_mod_insns_table_body
 // 16-bit encodings added by RVY+C
-:rvyc_ext_name:         Zyc
-:rvyc_file_name:        {rvyc_ext_name}_insns_table_body
-:rvyc_xref:             <<{rvyc_file_name}, {rvyc_ext_name}>>
+:rvy_c_ext_name:           "C" Extension ({cheri_base_ext_name})
```

Comment:
```suggestion
:rvy_c_ext_name:           "Zca" Extension ({cheri_base_ext_name})
```

Reply:
I think we should do a find and replace on "C or Zca, RVY" with "Zca, RVY".
Zca is a subset of C, so it's still accurate and less confusing.